prov/efa/test/efa_unit_test_info.c

```diff
@@ -94,24 +94,62 @@ void test_info_dgram_attributes()
 /**
  * @brief Verify that efa direct path fi_info objects have some expected values
  */
-void test_info_direct_attributes()
+static void test_info_direct_attributes_impl(struct fi_info *hints,
+					     int expected_return)
 {
-	struct fi_info *hints, *info = NULL, *info_head = NULL;
+	struct fi_info *info = NULL, *info_head = NULL;
 	int err;
 
-	hints = efa_unit_test_alloc_hints(FI_EP_RDM, EFA_DIRECT_FABRIC_NAME);
-	assert_non_null(hints);
+	err = fi_getinfo(FI_VERSION(1, 6), NULL, NULL, 0, hints, &info_head);
+	assert_int_equal(err, expected_return);
+	if (expected_return)
+		return;
 
-	err = fi_getinfo(FI_VERSION(1,6), NULL, NULL, 0, hints, &info);
-	assert_int_equal(err, 0);
-	assert_non_null(info);
+	assert_non_null(info_head);
 	for (info = info_head; info; info = info->next) {
-		assert_true(!strcmp(info->fabric_attr->name, EFA_DIRECT_FABRIC_NAME));
+		assert_true(!strcmp(info->fabric_attr->name,
+				    EFA_DIRECT_FABRIC_NAME));
 		assert_true(strstr(info->domain_attr->name, "rdm"));
 		assert_false(info->caps & (FI_ATOMIC | FI_TAGGED));
 		assert_false(info->tx_attr->msg_order & FI_ORDER_SAS);
-		assert_int_equal(info->ep_attr->max_msg_size, g_device_list[0].max_rdma_size);
+		assert_int_equal(
```

Comment:
Could you also add
```
assert_int_equal(info->domain_attr->progress, FI_PROGRESS_AUTO);
assert_int_equal(info->domain_attr->control_progress, FI_PROGRESS_AUTO);
```

similar to https://github.com/ofiwg/libfabric/commit/103f94f24529ec3022716eb889f9a387aa9de359

Reply:
Done